### DISCUSSION TITLE
first person to reply will be marked as answer | 🧠 Galaxy brain 🧠

### DISCUSSION THREAD

**Discussion by LarsPeterson** [Schweinepriester/github-profile-achievements#578] [General]
This is a discussion

**Comment by LarsPeterson** [comment]
@RealMrHex

**Comment by RealMrHex** [comment]
@LarsPeterson Just tap on the Gear emoji ⚙️ on the top right of discussion and change the category to QA
Then select my answer :)

**Reply by schambig** [reply]
Is this a recurrent activity? I would like to get the `Galaxy Brain` badge, can I turn on the notifications somewhere to get noticed next time? Thanks for you answer :D

**Comment by RealMrHex** [comment]
@schambig sure, ill create a QA for u
